### PR TITLE
Fix: don't update version file when publish alpha/beta version

### PR DESCRIPTION
**Pull request by chivalryq**
Signed-off-by: qiaozp <chivalry.pp@gmail.com>

### Description of your changes

When release contains "alpha" or "beta", don't update version pointer file in OSS.

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by barnettZQG — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -121,6 +121,7 @@ jobs:
         run: ./ossutil --config-file .ossutilconfig sync ./_bin/vela oss://$BUCKET/binary/vela/${{ env.VELA_VERSION }}
       
       - name: sync the latest version file
+        if: !contains(env.VELA_VERSION,'alpha') && !contains(env.VELA_VERSION,'beta')
```
Comment (barnettZQG) [inline .github/workflows/release.yml:124]: If could compare with remote value, it is better. otherwise, 1.3.6 maybe replace the 1.4.0.

**Comment by wonderflow** [comment]
@chivalryq I'll merging this  as it fixes the current bug for releasing beta version.  Already created a new issue to record this left issue.

File: .github/workflows/release.yml (hunk 1 above)
Comment (chivalryq) [reply .github/workflows/release.yml:124]: This is another problem, let's track it here: https://github.com/kubevela/kubevela/issues/3994

**Pull request merged**